SpringProject/src/it/java/com/gradproject2019/user/UserControllerIT.java

```diff
@@ -176,6 +180,36 @@ public void shouldReturn200WhenGettingEmployeeById() throws URISyntaxException {
         Assert.assertEquals(employee.getOccupation(), response.getBody().getOccupation());
     }
 
+    @Test
+    public void shouldReturn200WhenGettingEmployeesByUserId() {
+        ResponseEntity<List<UserResponseDto>> response = getUserByManagerId(uri3);
+
+        Assert.assertEquals(200, response.getStatusCodeValue());
+        Assert.assertEquals(response.getBody().size(), 1);
+        Assert.assertEquals(savedUserWithManager.getId(), response.getBody().get(0).getId());
+        Assert.assertEquals(savedUserWithManager.getFirstName(), response.getBody().get(0).getFirstName());
```

Comment:
I would argue these extra assertions don't add any value, just checking the ID is the same should be fine (correct me if I'm wrong)

Reply:
haha you told me to add extra assertions!!
tbf i only had the 200 assertion at the time so if id is all that needed to be added i'll remove the others